linkerd/service-profiles/src/client.rs

```diff
@@ -118,16 +134,19 @@ where
     S: GrpcService<BoxBody> + Clone + Send + 'static,
     S::ResponseBody: Send,
     <S::ResponseBody as Body>::Data: Send,
+    <S::ResponseBody as HttpBody>::Error:
+        Into<Box<dyn std::error::Error + Send + Sync + 'static>> + Send,
```

Comment:
I think this can just be `Into<Error>`?

Reply:
Ah, you're right — I had a memory of that not working, but I think that was something else, this compiles as you suggested.